msmbuilder/cluster/__init__.py

```diff
@@ -72,11 +72,5 @@ class SpectralClustering(MultiSequenceClusterMixin, cluster.SpectralClustering,
     __doc__ = _replace_labels(cluster.SpectralClustering.__doc__)
 
 
-class AgglomerativeClustering(MultiSequenceClusterMixin,
```

Comment:
I couldn't find any place where it was actually referenced. No init or doc, e.g.

```
[brooke:~/source/msmbuilder/msmbuilder/cluster (command-line)]$ grep AgglomerativeClustering *
[brooke:~/source/msmbuilder/msmbuilder/cluster (command-line)]$ grep SpectralClustering *
__init__.py:           'GMM', 'SpectralClustering', 'KCenters', 'NDGrid',
__init__.py:class SpectralClustering(MultiSequenceClusterMixin, cluster.SpectralClustering,
__init__.py:    __doc__ = _replace_labels(cluster.SpectralClustering.__doc__)
Binary file __init__.pyc matches
```